index.bs

```diff
@@ -1078,17 +1079,26 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     The [=credential private key=] is bound to a particular [=authenticator=] - its [=managing authenticator=] -
     and is expected to never be exposed to any other party, not even to the owner of the [=authenticator=].
 
-    Note that in the case of [=self
-    attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
-    for details.
+    Note: In the case of [=self attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=] for details.
 
     Note: The [=credential public key=] is referred to as the [=user public key=] in FIDO UAF [[UAFProtocol]], and in FIDO U2F
-    [[FIDO-U2F-Message-Formats]] and some parts of this specification that relate to it.
+    [[FIDO-U2F-Message-Formats]] and some parts of this specification that relate to it. Also, the term [=user credential=] is occasionally used to synonymously refer to [=credential public key=] and [=user public key=].
 
 : <dfn>Credential Properties</dfn>
 :: A [=credential property=] is some characteristic property of a [=public key credential source=], such as whether it is a
     [=client-side discoverable credential=] or a [=server-side credential=].
 
+: <dfn>Hardware-bound Device Key Pair</dfn>
+: <dfn>Device-bound Key</dfn>
+: <dfn>Device Private Key</dfn>
+: <dfn>Device Public Key</dfn>
+: <dfn>Device Key</dfn>
+: <dfn>Secondary Key</dfn>
```

Comment:
I suggest we wait until this PR is ready-to-merge and then, if appropriate, trim these terms?  I stuck all those in there (for now) because they were being variously used in discussions.

Reply:
Reading through this again I have some more thoughts... :smile: I'm thinking maybe it would be useful to generalize these terms to being about the "hardware-bound" property in general, not necessarily limited to the context of the `devicePubKey` extension. Then we could contrast that against the "synced key" term we've been thinking of adding. This paragraph also seems to rehash a lot of what's already said in the description of the extension, and if you take that out I think that would leave pretty much just the general concept. Thoughts on that?

Reply:
That sounds reasonable. how about this:

: <dfn>Hardware-bound Device Key Pair</dfn>
: <dfn>Device-bound Key</dfn>
: <dfn>Device Private Key</dfn>
: <dfn>Device Public Key</dfn>
:: A [=hardware-bound device key pair=], also known as a [=device-bound key=], is an [=authenticator=]-, [=[RP]=]-, and [=user credential=]-specific public key pair created upon a [=[RP]=]'s request via the [=devicePubKey=] [=WebAuthn extension=].
    The [=authenticator=] that a [=hardware-bound device key pair=] is created upon guarantees that the [=device private key=] is securely stored in hardware, i.e., it is unextractable. See [[#sctn-device-publickey-extension]].

..?

Reply:
I'd probably make it "see also" at the end, but that looks good to me!

Reply:
done in e1e6d94